src/components/vr_cooperation/include/vr_cooperation/commands/json_notification.h

```diff
@@ -51,23 +51,22 @@ namespace commands {
 /**
  * @brief Base command class for notifications
  */
-class BaseCommandNotification : public Command {
+class JSONNotification : public Command {
  public:
   /**
    * @brief BaseCommandNotification class constructor
```

Comment:
Please change BaseCommandNotification to JsonNotitication everywhere in this class